source/content/guides/filesystem/03-large-files.md

```diff
@@ -49,13 +54,13 @@ The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is
 |:----------------|--------------------------------------|-------------------------------------|
 | ≤ 100MiB        | <span  style="color:green">✔</span>  | Can be uploaded via any means       |
 | 100MiB - 256MiB | <span  style="color:orange">✔</span> | Must be uploaded over SFTP or rsync |
-| > 256MiB        | <span  style="color:red">❌</span>    | Must be hosted via 3rd-party CDN    |
+| > 256MiB        | <span  style="color:red">❌</span>    | Must be offloaded to an external filesystem|
```

Comment:
```suggestion
| > 256MiB        | <span  style="color:red">❌</span>    | Must be offloaded to an external filesystem
```
(minor) We aren't terminating other rows with `|`; recommend staying consistent.